csrankings.csv

```diff
@@ -9358,6 +9358,7 @@ Markus Wagner 0007,University of Adelaide,https://cs.adelaide.edu.au/~markus,DVb
 Marleen de Bruijne,University of Copenhagen,http://image.diku.dk/marleen,7kTk3LUAAAAJ
 Marlon Dumas,University of Tartu,http://math.ut.ee/~dumas,9lIttRkAAAAJ
 Marnel Peradilla,De La Salle University,https://www.dlsu.edu.ph/colleges/ccs/faculty-profile/single/?id=32665800576&command=GETPROFILE,Cc5_76kAAAAJ
+Marouane Kessentini,University of Michigan,http://kessentini.net,5oW_MA8AAAAJ

```

Comment:
The University of Michigan in the database is the Ann Arbor campus, not any other campus.